docs/howto/debugging.md

```diff
@@ -81,6 +82,13 @@ See also [in the "Troubleshooting" section below](#debug-remotely-never-loads).
 [chrome-devtools-device]: https://facebook.github.io/react-native/docs/debugging.html#debugging-on-a-device-with-chrome-developer-tools
 
 
+<div id="react-devtools">
+
+## React Devtools
+
+The standalone version of React DevTools, which is an electron app, can be used to inspect the react-native components of the app. Since the version of react-native we use is `0.59`, the latest version (v4) of react-devtools is not supported, and v3 needs to be installed.  This can be done using the command `npm i -g react-devtools@^3` See [this blog post](https://reactjs.org/blog/2019/08/15/new-react-devtools.html#which-versions-of-react-are-supported) for complete compability information. Refer to [it's npm page](https://www.npmjs.com/package/react-devtools) for instructions on how to use with React Native.
```

Comment:
Rather than the `npm` CLI, we use `yarn`. For consistency, in this doc let's recommend `yarn global add` rather than `npm i -g`.